conan/cli/formatters/report/diff.py

```diff
@@ -0,0 +1,99 @@
+import json
+import os
+import base64
+
+from jinja2 import Template
+
+from conan.api.output import cli_out_write
+from conan.cli.formatters.report.diff_html import diff_html
+
+def _generate_json(result):
```

Comment:
The idea is the the html generator uses the json representation as a source for its contents, so that we can just pre-process it once